ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/PutJobManagement_Test.java

```diff
@@ -246,8 +246,9 @@ public void truncateJobCancelWithOutForce() throws Exception {
             client.putObject(new PutObjectRequest(BUCKET_NAME, book1, book1Channel, jobId, 0, Files.size(objPath1)));
             ABMTestHelper.waitForJobCachedSizeToBeMoreThanZero(jobId, client, 20);
 
-            final CancelJobSpectraS3Response failedResponse = client.cancelJobSpectraS3(new CancelJobSpectraS3Request(jobId.toString()));
-            assertThat(failedResponse.getStatusCode(),is(400));
+            final TruncateJobSpectraS3Response failedResponse = client.truncateJobSpectraS3(
```

Comment:
good point . . . it used to return a 400, so I should update the variable name as well.